typesafe_conductr_cli/test/cli_test_case.py

```diff
@@ -0,0 +1,42 @@
+from unittest.mock import MagicMock
+from requests.exceptions import ConnectionError, HTTPError
+
+
+class CliTestCase():
+    """Provides test case common functionality"""
+
+    @property
+    def default_connection_error(self):
+        return self.strip_margin("""|ERROR: Unable to contact Typesafe ConductR.
+                                    |ERROR: Reason: test reason
+                                    |ERROR: Make sure it can be accessed at {}:{}.
+                                    |""")
+
+    def respond_with(self, status_code=200, text=""):
+        reasons = {
+            200: "OK",
+            404: "Not Found"
+        }
+
+        response_mock = MagicMock(
+            status_code=status_code,
+            text=text,
+            reason=reasons[status_code])
+
+        if status_code == 200:
+            response_mock.raise_for_status.return_value = False
```

Comment:
I'm not really up to speed on Python, but it looks like setting this to `None` is more common than `False`. Is it just a matter of taste?